core/nwb.ecephys.yaml

```diff
@@ -229,8 +229,16 @@ groups:
   - name: location
     dtype: text
     doc: Location of electrode group. Specify the area, layer, comments on estimation
-      of area/layer, stereotaxic coordinates if in vivo, etc. Use standard atlas
-      names for anatomical regions when possible.
+      of area/layer, etc. Use standard atlas names for anatomical regions when possible.
+  datasets:
+  - name: position
+    dtype: float
+    shape:
+    - 3
+    dims:
+    - x, y, z
```

Comment:
Rather than having a 1-d array of length 3, I would use either three scalar datasets for x, y, and z (like in the electrodes table, for consistency), or a compound data type for x, y, and z, which is what we wished we had done for the electrodes table.
```yaml
dtype:
- name: x
  dtype: float
  doc: x coordinate
- name: y
  dtype: float
  doc: y coordinate
- name: z
  dtype: float
  doc: z coordinate
```